Gemfile.lock

```diff
@@ -92,7 +92,6 @@ DEPENDENCIES
   jekyll-sitemap
   tzinfo (~> 1.2)
   tzinfo-data
-  wdm (~> 0.1.1)
```

Comment:
Please avoid committing this as it breaks the build. Is there a problem you're having with it? I don't use Windows, so it's not bothering me, but I also don't know how useful it is.